package.json

```diff
@@ -38,8 +42,8 @@
     "dialog-polyfill": "^0.4.10",
     "generator-rhelement": "^0.6.4",
     "global": "^4.3.2",
-    "gulp": "github:gulpjs/gulp#4.0",
-    "gulp-babel": "^7.0.0",
+    "gulp": "^4.0.0",
```

Comment:
@mwcz Great! I didn't realize that Gulp 4.0 was out. Looks like I'm pretty far behind. It was released in December. https://github.com/gulpjs/gulp/releases

Reply:
I was happy to see that too!